### PR TITLE
Add RSS/Atom feed

### PR DESCRIPTION
**Pull request by itrich**
This PR adds a Atom feed under `/feed.xml` as well as two sub-feeds located at `/feed/security.xml` and `/feed/release.xml`.

### CODE REVIEW

File: _data/social.yml [hunk 1]
```diff
@@ -1,15 +1,18 @@
 - title: Twitter
-  icon: fa-twitter
+  icon: fa-twitter fa-lg
```
Comment (fkr) [inline _data/social.yml:2]: the changes to adjust the size of the icons are part of this changeset on purpose?

File: _i18n/de/_posts/announcements/2021-07-15-release0.md [hunk 2]
```diff
@@ -12,7 +13,7 @@ haben, haben wir dennoch bereits eine Community aufgebaut, welche nützliche
 Arbeit durch gemeinschaftliche Software Entwicklung leisten konnte.
 
 R0 ist das Ergebnis von mehr als einem Jahr Arbeit, welche alle auf github
-nachvollziehbar ist. 
+nachvollziehbar ist.
```
Comment (fkr) [inline _i18n/de/_posts/announcements/2021-07-15-release0.md:16]: Various not-related whitespace changes present.

File: _includes/navigation.html [hunk 3]
```diff
@@ -39,7 +39,7 @@
         {% for item in site.data.social %}
         <li class="nav-item my-auto mx-2 mx-lg-0">
           <a class="nav-link" target="_blank" href="{{ item.link }}">
-            <i class="fa {{item.icon}} fa-lg"></i>
+            <i class="fa {{item.icon}}"></i>
```
Comment (fkr) [inline _includes/navigation.html:42]: layout change (size of icon) intended for this changeset?

**Review by fkr — APPROVED** [review]

File: _data/social.yml (hunk 1 above)
Comment (itrich) [reply _data/social.yml:2]: I originally had this in another branch but got messed up with the branches. That's why both features are within this PR.

File: _i18n/de/_posts/announcements/2021-07-15-release0.md (hunk 2 above)
Comment (itrich) [reply _i18n/de/_posts/announcements/2021-07-15-release0.md:16]: I should configure my IDE to actually stop doing this... :angry:

**Pull request merged**